### ISSUE TITLE
[WPF] Drop downs’ border in Settings->application has insufficient contrast against its adjacent color

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#878]
The following AI-windows mode(s) have this issue:
- Regular mode
- High Contrast White mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![25 1](https://user-images.githubusercontent.com/22068434/88357325-bace8b80-cd1f-11ea-89fa-035c3dd21d9c.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by juchampa** [comment]
This is a WPF specific issue
